### ISSUE TITLE
GIF 

### ISSUE THREAD

**Issue by uraabk** [ghtx280/imgen#1]
Can you add a GIF format?

**Comment by ghtx280** [comment]
yes, this is in plans, but it will probably load hosting, I use a free one, so maybe this feature will be paid, idk(
